java-ast/tests/pom.xml

```diff
@@ -41,6 +41,12 @@
             <artifactId>windup-exec</artifactId>
             <classifier>forge-addon</classifier>
             <scope>test</scope>
+        </dependency>    
+        <dependency>
+            <groupId>org.eclipse.tycho</groupId>
+            <artifactId>org.eclipse.jdt.core</artifactId>
+            <version>3.10.0.v20140604-1726</version>
+            <scope>test</scope>
```

Comment:
Thanks @mareknovotny. Do you know which Maven version is CI using? In my case it's 3.3.9.